members/models/member.py

```diff
@@ -61,6 +61,14 @@ def get_current_membership_period(self, current_date: datetime.date = None):
     def get_periods(self):
         return self.membershipperiod_set.all().order_by('start_date')
 
+    def get_email(self):
+        if self.primary_email_in_use:
+            return self.primary_email
+        elif self.secondary_email_in_use:
+            return self.secondary_email
+        else:
+            return self.primary_email
```

Comment:
Why send emails to members with no active email address?

Reply:
good point i misread the issue about that so i fixed it